CHANGELOG.rst

```diff
@@ -16,6 +16,12 @@ Change Log
 Unreleased
 ----------
 
+[3.17.48]
```

Comment:
nit: might recommend bumping to a new feature release (3.18.0) as this is net-new functionality?